src/libData/AccountData/AccountStoreSC.tpp

```diff
@@ -1128,6 +1128,7 @@ bool AccountStoreSC<MAP>::ParseCallContractJsonOutput(
     if (ret) {
       if (!contractAccount->SetStorage(state_entries, temp)) {
         LOG_GENERAL(WARNING, "SetStorage failed");
+        return false;
```

Comment:
Similar to this, is a `return false` needed on line 1144 as well? (GitHub doesn't let me comment on that line directly).